ui/raidboss/data/04-sb/trial/lakshmi-ex.js

```diff
@@ -43,43 +41,30 @@
       },
     },
     {
-      id: 'Lakshmi Pull of Light',
+      id: 'Lakshmi Pull of Light on Tank',
```

Comment:
For consistency with other triggers, just call this `Lakshmi Pull Of Light Tank` and `Lakshmi Pull Of Light Unexpected`?